api/envoy/api/v2/lds.proto

```diff
@@ -98,7 +98,7 @@ message Listener {
   }
 
   // [#not-implemented-hide:]
-  DeprecatedV1 deprecated_v1 = 7;
+  DeprecatedV1 deprecated_v1 = 7 [deprecated = true];
```

Comment:
I think there are more than this as @rshriram points out? I'm pretty sure we have a bunch of these...

Reply:
I believe the rest were already deprecated :-)
I'm going to close this out regardless though - I'd thought updating the deprecation notice from last release to this release would make deprecation schedule more clear but if we'd prefer to not rewrite history I don't think deprecating it every version until we remove is helpful